### PR TITLE
fix: update data visualization section

### PR DESCRIPTION
**Pull request by pa7ri**
**Closes Issue #1110**- Revise Check for Understanding section in data visualization

1. Following CFU guide #700, re-write questions with topics covered in this section
2. Revise "Let's Make a Bar Chart" section. _NOTE: Most of suggested changes in [#1086](https://github.com/Techtonica/curriculum/issues/1086#issuecomment-567664466) where solved in #1107)_ 

**In addition**
Unify punctuation marks for every paragraph in data visualization section

### CODE REVIEW

File: data-visualization/data-visualization.md [hunk 1]
```diff
@@ -25,34 +25,34 @@ Companies like [Fidelity](https://www.fidelity.com/) and [Strava](https://www.st
 
 **Participants will be able to:**:
 
-- Become more familiar with publically available data sets
-- Gain some experience determining how to select data to visualize
-- Learn about several common tools and open-source libraries for visualization
-- Utilize one (or several) tools to display data in a compelling and meaningful way
-- Use D3 methods in JavaScript to create DOM elements
-- Learn about SVG and its elements
+- Become more familiar with publically available data sets.
```
Comment (alodahl) [inline data-visualization/data-visualization.md]: I don't know if I agree with this change... these aren't actually complete sentences.  It's ok if bulleted lists have no punctuation, i just has to be consistent either way.  

Please change these back. Liens 37 & 38 too, please.

File: data-visualization/data-visualization.md [hunk 2]
```diff
@@ -223,9 +223,14 @@ const g = d3.select("svg")
 
 ### Check for Understanding
 
-- Discuss how one would go about updating the DOM and utilizing D3.js. What are the general steps to set up a new D3.js visualization?
-- What factors do you consider when selecting a data set?
-- What do you consider when determining how to display data in a compelling way?
+- Which tools and open-source libraries are used for data visualization?
+- Which imports and methods from D3.js would you use to set up a new D3.js visualization or update the DOM?
+- What attributes are customizable for a chart or graph? List some.
+Form small groups and discuss:
```
Comment (alodahl) [inline data-visualization/data-visualization.md]: This doesn't actually end up on a new line in markdown.  Look at the markdown preview to make sure it looks as expected.

**Comment by alodahl** [comment]
BTW, I love your conventional commit title 🤓

**Comment by pa7ri** [comment]
Thanks for your review @alodahl. I'll keep these comments in mind for following contributions :smile:

**Comment by Kundan28** [comment]
Updated in scoresheet.

**Pull request merged**